sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -58,39 +58,170 @@ and they must be included in the order shown:
-|``` make<event>(const context &, ze_event_handle_t, ownership = transfer);```| Constructs a SYCL event instance from a Level-Zero ```ze_event_handle_t```. The context argument must be a valid SYCL context encapsulating a Level-Zero context. The Level-Zero event should be allocated from an event pool created in the same context. The ```ownership``` argument specifies if the SYCL runtime should take ownership of the passed native handle. The default behavior is to transfer the ownership to the SYCL runtime. See section 4.4 for details.|
-|``` make<program>(const context &, ze_module_handle_t);```| Constructs a SYCL program instance from a Level-Zero ```ze_module_handle_t```. The context argument must be a valid SYCL context encapsulating a Level-Zero context. The Level-Zero module must be fully linked (i.e. not require further linking through [```zeModuleDynamicLink```](https://spec.oneapi.com/level-zero/latest/core/api.html?highlight=zemoduledynamiclink#_CPPv419zeModuleDynamicLink8uint32_tP18ze_module_handle_tP28ze_module_build_log_handle_t)), and thus the SYCL program is created in the "linked" state.|
+The following free functions defined in the ```sycl``` namespace are specialized for Level-Zero backend to allow
+an application to create a SYCL object that encapsulates a corresponding Level-Zero object:
+
+<table>
+<tr>
+<th>Level-Zero interoperability function</th>
+<th style="text-align:left"> Description</th>
+</tr><tr>
+<td>
+
+``` C++
+make_platform<backend::ext_oneapi_level_zero>(
+    const backend_input_t<
+        backend::ext_oneapi_level_zero, platform> &)
+```
+</td>
+<td>Constructs a SYCL platform instance from a Level-Zero <code>ze_driver_handle_t</code></td>
+</tr><tr>
+<td>
+
+``` C++
+make_device<backend::ext_oneapi_level_zero>(
+    const backend_input_t<
+        backend::ext_oneapi_level_zero, device> &)
+```
+</td>
+<td>Constructs a SYCL device instance from a Level-Zero <code>ze_device_handle_t</code>.</td>
+</tr><tr>
+<td>
+
+``` C++
+make_context<backend::ext_oneapi_level_zero>(
+    const backend_input_t<
+        backend::ext_oneapi_level_zero, context> &)
+```
+</td>
+<td>Constructs a SYCL context instance from a Level-Zero <code>ze_context_handle_t</code>. The context is created against the devices passed in <code>DeviceList</code> structure member. There must be at least one device given and all the devices must be from the same SYCL platform and thus from the same Level-Zero driver. The <code>Ownership</code> input structure member specifies if the SYCL runtime should take ownership of the passed native handle. The default behavior is to transfer the ownership to the SYCL runtime. See section 4.4 for details.</td>

```

Comment:
> The default behavior is to transfer the ownership to the SYCL runtime.

How can there be a default behavior?  The API doesn't have a default value for the `Ownership` field because it is not possible to have a default value for one field in a structure like this.  Same comment for all the other API descriptions below.

Reply:
OK, except that it is also true for sub-devices